api/users.go

```diff
@@ -24,9 +24,12 @@ func getUsers(w http.ResponseWriter, r *http.Request) {
 
 func addUser(w http.ResponseWriter, r *http.Request) {
 	var user db.User
+
 	if err := mulekick.Bind(w, r, &user); err != nil {
 		return
 	}
+<<<<<<< HEAD
```

Comment:
unresolved merge conflict has been left in here

Reply:
Saw it, too. I just wasn't fast enough to push before you found it...

Reply:
😁 just happened to be around as you pushed, thanks for the fix. I'll try to work out why circle is failing at the moment